lint/src/test/java/com/firebaseui/lint/NonGlobalIdDetectorTest.kt

```diff
@@ -40,4 +42,15 @@ class NonGlobalIdDetectorTest {
                         |+     android:id="@+id/invalid"/>
                         |""".trimMargin())
     }
+
+    companion object {
+        // Nasty hack to make lint tests pass on Windows
+        private val sdkPath = System.getProperty("java.library.path").split(';').find {
+            it.contains("SDK", true)
```

Comment:
Can you add some more comments explaining how this hack works?

Reply:
Done.